CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.5.0](https://github.com/rewardops/rewardops-sdk-node/compare/v2.4.0...v2.5.0) (2021-03-02)
+
+### Features
+
+- filter sensitive data from logs MX-1252 ([#70](https://github.com/rewardops/rewardops-sdk-node/issues/70)) ([06e373e](https://github.com/rewardops/rewardops-sdk-node/commit/06e373eda51b2fa53c669f485786917fff8e8f19))
+- return headers for all api calls MX-1260 ([#69](https://github.com/rewardops/rewardops-sdk-node/issues/69)) ([95f3126](https://github.com/rewardops/rewardops-sdk-node/commit/95f3126e43e15ad5ee107510243d64e7d8b19c65))
```

Comment:
Given this is an open-source library, I've been removing the references to Jira tickets from the CHANGELOG
```suggestion
- filter sensitive data from logs ([#70](https://github.com/rewardops/rewardops-sdk-node/issues/70)) ([06e373e](https://github.com/rewardops/rewardops-sdk-node/commit/06e373eda51b2fa53c669f485786917fff8e8f19))
- return headers for all api calls ([#69](https://github.com/rewardops/rewardops-sdk-node/issues/69)) ([95f3126](https://github.com/rewardops/rewardops-sdk-node/commit/95f3126e43e15ad5ee107510243d64e7d8b19c65))
```